### PR TITLE
[Feature/AES-1346]: Update Content Hub Related Components

### PR DESCRIPTION
**Pull request by zack76**
Remove Success Update message for Load More Button
Add hover effects for content hub article image

- Image grayscale

- underline the title when hovering on image
Screen Shot 2020-07-31 at 11 34 32 am

Add data-test-ref for Load More Button and Content Hub Article

### CODE REVIEW

**Comment by beau-charman** [comment]
I was thinking about this one, as I mentioned in the meeting we has a few weeks, this load more button is simple and didnt need to be as complex as the Add ToCard button, it doesnt need the slide styles and seperate divs. Please update so the content of the button simply updates. 

```
<Button ... >{isLoading ? <Loading {...} /> ? buttonLabel}</Button>
```

File: src/components/ContentHubArticle/ContentHubArticle.js [hunk 1]
```diff
@@ -26,60 +27,109 @@ const ContentHubArticle = ({
   const currentImage = isHorizontal ? horizontalThumbnail : verticalThumbnail;
   const classSet = cx(styles.base, className);
   const categoryClassSet = cx(styles.category);
-  const imageClassSet = cx(styles.link);
-  const nonMobileImageClassSet = cx(styles.link, styles[`non-mobile`]);
+  const imageClassSet = cx(styles.image);
+  const nonMobileImageClassSet = cx(styles.image, styles[`non-mobile`]);
```
Comment (beau-charman) [inline src/components/ContentHubArticle/ContentHubArticle.js:31]: keep the classNames camel case consistent. `nonMobile`

File: src/components/ContentHubArticle/ContentHubArticle.module.css [hunk 2]
```diff
@@ -38,6 +40,16 @@
   border-bottom: 1px solid var(--color-secondary);
 }
 
+.title-active {
```
Comment (beau-charman) [inline src/components/ContentHubArticle/ContentHubArticle.module.css:43]: same here, `titleActive`

File: src/components/ContentHubArticle/ContentHubArticle.js (hunk 1 above)
Comment (zack76) [reply src/components/ContentHubArticle/ContentHubArticle.js:31]: update it now

**Pull request merged**